src/app/state/api.js

```diff
@@ -2,11 +2,12 @@
 
 const url = '__BASE_DOMAIN__/api/v1/data/';
 
-export async function getProbeData(params) {
+export async function getProbeData(params, token) {
   const data = await fetch(url, {
     method: 'POST',
     headers: {
       'Content-Type': 'application/json',
+      Authorization: `Token ${token}`,
```

Comment:
This should be a `Bearer` token. The backend is looking for that string.

Reply:
Thanks!